### PR TITLE
kubevirtci: Bump version

### PR DESCRIPTION
**Pull request by oshoval**
**What this PR does / why we need it**:
Bump kubevirtci and cleanup unrequired exports.

Allow pinning specific kubevirtci per lane.
It will allow decoupling kubevirtci tag version between the lanes.

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by oshoval** [comment]
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/1807/pull-e2e-cluster-network-addons-operator-ovs-cni-functests/1799769689681825792
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/1807/pull-e2e-cluster-network-addons-operator-workflow-k8s/1799769689086234624

Those are failing, rerunning just to see if it is flake or static

/retest

**Comment by oshoval** [comment]
closing
we will try to use github actions

this need to be fixed on a follow-up orthogonal effort

**Comment by oshoval** [comment]
@phoracek 
ptal, as we spoke offline

File: automation/check-patch.e2e-lifecycle-k8s.sh
Comment (oshoval) [inline automation/check-patch.e2e-lifecycle-k8s.sh:21]: we can even drop this line (and the one in the next file)
as it is the default anyhow taken from cluster.sh

**Comment by oshoval** [comment]
need to fix
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/1807/pull-e2e-cluster-network-addons-operator-kubemacpool-functests/1808394046276636672

/hold

File: automation/check-patch.e2e-ovs-cni-functests.sh
Comment (phoracek) [inline automation/check-patch.e2e-ovs-cni-functests.sh:23]: Mark this with TODO and link the tracker of the issue

File: automation/check-patch.e2e-lifecycle-k8s.sh
Comment (phoracek) [reply automation/check-patch.e2e-lifecycle-k8s.sh:21]: Please drop it

File: automation/components-functests.setup.sh
Comment (phoracek) [inline automation/components-functests.setup.sh:102]: Either remove this or provide enough context to make this actionable

File: automation/components-functests.setup.sh
Comment (phoracek) [inline automation/components-functests.setup.sh:103]:
```suggestion
if [[ $COMPONENT == "kubemacpool" ]]; then
```

IIUIC the statement you used would always return true

**Review by phoracek — CHANGES_REQUESTED** [review]

File: automation/components-functests.setup.sh
Comment (oshoval) [reply automation/components-functests.setup.sh:103]: the PR is WIP / draft as i debug something all those will be dropped
not ready to review (saw the failure just after i asked sorry)
thanks

File: automation/check-patch.e2e-ovs-cni-functests.sh
Comment (oshoval) [reply automation/check-patch.e2e-ovs-cni-functests.sh]: found the root cause it seems, will try to fix it from the root

**Comment by oshoval** [comment]
hold would be removed once the last commit is dropped, atm it is there just to see the fixes it poced
helps

File: automation/components-functests.setup.sh
Comment (oshoval) [reply automation/components-functests.setup.sh]: It will be dropped before this PR is merged, just poced to see it fix the root problem

File: automation/components-functests.setup.sh
Comment (oshoval) [reply automation/components-functests.setup.sh]: https://github.com/kubevirt/cluster-network-addons-operator/pull/1807#discussion_r1664287961

File: automation/check-patch.e2e-ovs-cni-functests.sh
Comment (oshoval) [reply automation/check-patch.e2e-ovs-cni-functests.sh]: dropped since the root cause is fixed (need to merge ovs and kmp PRs for that)

File: automation/check-patch.e2e-lifecycle-k8s.sh
Comment (oshoval) [reply automation/check-patch.e2e-lifecycle-k8s.sh]: done, lets see all pass

File: automation/components-functests.setup.sh
Comment (oshoval) [reply automation/components-functests.setup.sh]: removed, lets see all pass (required PRs were merged on OVS/KMP)

**Comment by oshoval** [comment]
comments were addressed, all debug code was remove, and at the end we don't need to pin OVS
to an old version, as the root cause was fixed
(wrong kubevirtci tag was used when running OVS e2e tests,
the right tag should be according the actual deployed cluster, i.e the one CNAO uses)

**Comment by oshoval** [comment]
all passed, addressed comments,
now the PR has the minimal trivial required changes for a bump

/lgtm
/approve

**Pull request merged**
